components/brave_wallet_ui/components/desktop/popup-modals/add-account-modal/hardware-wallet-connect/index.tsx

```diff
@@ -219,14 +219,15 @@ export const HardwareWalletConnect = ({ onSuccess, selectedAccountType }: Props)
             >
               <LedgerIcon />
             </HardwareButton>
-
-            <HardwareButton
-              onClick={onSelectTrezor}
-              isSelected={selectedHardwareWallet === BraveWallet.TREZOR_HARDWARE_VENDOR}
-              disabled={isConnecting && selectedHardwareWallet !== BraveWallet.TREZOR_HARDWARE_VENDOR}
-            >
+            {(selectedAccountType.coin === BraveWallet.CoinType.ETH) &&
+              <HardwareButton
+                onClick={onSelectTrezor}
+                isSelected={selectedHardwareWallet === BraveWallet.TREZOR_HARDWARE_VENDOR}
+                disabled={isConnecting && selectedHardwareWallet !== BraveWallet.TREZOR_HARDWARE_VENDOR}
+              >
               <TrezorIcon />
-            </HardwareButton>
+              </HardwareButton>
+            }
```

Comment:
I made an attempt to hide the 'Trezor' button for anything other than CoinType::ETH here, but it results in the 'Ledger' button being offset to the left.  Probably need advice from frontend team on how to handle.
![image](https://user-images.githubusercontent.com/12549658/173113518-6dbadcb7-9cb5-4e55-b989-785bbf85c5e6.png)

Reply:
Maybe we should hide both `Select your hardware wallet device` title and `Ledger` button if the `cointType` is `SOL` and then replace it with just the `Ledger` logo?

Reply:
I didn't manage to add the ledger logo because the spacing was a little funny - 
![image](https://user-images.githubusercontent.com/12549658/174115233-6f285813-53c1-49e8-9c41-bedb78d51e5b.png)


Otherwise I achieved parity with what we are doing currently for filecoin (which also only supports Ledger).  It looks like this now:
![image](https://user-images.githubusercontent.com/12549658/174115405-1e903450-55f7-4e95-86a7-f57c3d26c69b.png)

Reply:
Perfect!